dirty_cat/gap_encoder.py

```diff
@@ -50,12 +57,28 @@ class GapEncoderColumn(BaseEstimator, TransformerMixin):
 
     """See GapEncoder's docstring."""
 
-    def __init__(self, n_components=10, batch_size=128, gamma_shape_prior=1.1,
-                 gamma_scale_prior=1.0, rho=.95, rescale_rho=False,
-                 hashing=False, hashing_n_features=2**12, init='k-means++',
-                 tol=1e-4, min_iter=2, max_iter=5, ngram_range=(2, 4),
-                 analyzer='char', add_words=False, random_state=None,
-                 rescale_W=True, max_iter_e_step=20):
+    rho_: float
+    H_dict_: Dict[np.array, np.array]
+
+    def __init__(self,
+                 n_components: int = 10,
+                 batch_size: int = 128,
+                 gamma_shape_prior: float = 1.1,
+                 gamma_scale_prior: float = 1.0,
+                 rho: float = .95,
```

Comment:
It used to be (in PEP8) that for keyword arguments, there should not be a space before and after the equal sign.

Can you check that the present formatting is valid with style guidelines?

Reply:
Yes, this is valid :)